crates/bevy_asset/src/filesystem_watcher.rs

```diff
@@ -2,8 +2,9 @@ use crossbeam_channel::Receiver;
 use notify::{Event, RecommendedWatcher, RecursiveMode, Result, Watcher};
 use std::path::Path;
 
-/// Watches for changes to assets on the filesystem. This is used by the `AssetServer` to reload
-/// them
+/// Watches for changes to files on the local filesystem.
+///
+/// This is used by the [`AssetServer`] to reload assets when their source paths are changed.
```

Comment:
it's not when the path changes, it's when the file changes

Reply:
Indeed, "changed" was supposed to be "modified".